ScratchWikiSkinTemplate.php

```diff
@@ -197,10 +217,10 @@ public function execute() {
 $link = "<a href=\"$url\" target=\"_blank\">$text</a>";
 $line = wfMessage('scratchwikiskin-dark-theme-feedback')->rawParams( $link )->inLanguage( $wgLang )->escaped();
 ?>
-					<div id="feet" style="margin: 0<?=$darkPref ? '' : '; display: none'?>"><?=$line?></div>
+					<div class="feet" style="margin: 0<?=$darkPref ? '' : '; display: none'?>"><?=$line?></div>
```

Comment:
This still doesn't work, since the CSS looks for `#feet`, not `.feet` - please make sure to test your code before committing it

Reply:
I'm sorry! I did change the CSS to `.feet`, but forgot to submit the updated CSS.